ArchUnitNET/Fluent/Syntax/Elements/Members/GivenMembersThat.cs

```diff
@@ -71,6 +71,11 @@ public TGivenRuleTypeConjunction AreStatic()
             return Create<TGivenRuleTypeConjunction, TRuleType>(_ruleCreator);
         }
 
+        public TGivenRuleTypeConjunction AreReadOnly()
+        {
+            _ruleCreator.AddPredicate(MemberPredicatesDefinition<TRuleType>.AreReadOnly());
+            return Create<TGivenRuleTypeConjunction, TRuleType>(_ruleCreator);
+        }
         //Negations
```

Comment:
This comment belongs to the next section. Please move it down a little, e.g. by adding a new line after the new method